<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blast Query++</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="search_label">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchtext">
          <property name="placeholderText">
           <string>Gene Name or Accession Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="accession_label">
          <property name="text">
           <string>Accession Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="accession_list">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_csv">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save to CSV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="path_label">
          <property name="styleSheet">
           <string notr="true">QLabel { color : rgb(179, 179, 179); }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="selection1"/>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="list1">
             <attribute name="title">
              <string>List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QTableWidget" name="list_table1"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="filtered1">
             <attribute name="title">
              <string>Filtered List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QTableWidget" name="filtered_table1"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="graph1">
             <attribute name="title">
              <string>Graph</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QWidget" name="plot1" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="selection2"/>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="list2">
             <attribute name="title">
              <string>List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QTableWidget" name="list_table2"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="filtered2">
             <attribute name="title">
              <string>Filtered List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QTableWidget" name="filtered_table2"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="graph2">
             <attribute name="title">
              <string>Graph</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QWidget" name="plot2" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="selection3"/>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_3">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="list3">
             <attribute name="title">
              <string>List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QTableWidget" name="list_table3"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="filtered3">
             <attribute name="title">
              <string>Filtered List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QTableWidget" name="filtered_table3"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="graph3">
             <attribute name="title">
              <string>Graph</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QWidget" name="plot3" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="gene_textbox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChoose_Work_Folder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChoose_Work_Folder">
   <property name="text">
    <string>Choose Work Folder...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
